sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1271,10 +1271,11 @@ class DataFrame private[sql](
    * @since 1.6.0
    */
   @scala.annotation.varargs
-  def drop(colNames: String*): DataFrame = {
+  def drop(colName: String, colNames: String*): DataFrame = {
```

Comment:
Never mind, the annotation is wrong it was added in #9862 which did not get merged into 1.6 so no worry about changing the method signature.